examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -11,6 +11,7 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
       sentimentUpperBound: 4,
       cacheThreshold: parseInt(config.cacheThreshold),
       querySliceMills: parseInt(config.querySliceMills),
+      pinMapOneTweetResult: null,
```

Comment:
Let's change the name to the following to make it more clear.

pinMapOneTweetLookupResult

Reply:
Sure. Thanks for your patience. Will change !

Reply:
Thanks. pinMapOneTweetLookUpResult